src/output-json-ike.c

```diff
@@ -0,0 +1,189 @@
+/* Copyright (C) 2020 Open Information Security Foundation
```

Comment:
Could you have first done `git mv output-json-ikev2.c output-json-ike.c `

Reply:
By the way, is there a set of bash commands like `sed` that you ran to get this commit ?

Reply:
Normally i'm resetting my commits and use meld and a second suricata-folder to create the different commits, ensuring that they compile. But it's pretty painful with the amount of changes....
Do you have a better approach to that?

Did you mean that? :)

Reply:
Well, I do not know if my approach would work in this case.
But I would have done 
one commit with `git mv "*ikev2*" "*ike*"` (and looking for the file names with git grep)
one commit like `find . -name "*.c" | sed -i -e s/ikev2/ike/g` (for the different files, and CamelCase and so on)

I do not have the exact syntax, but I would have tried something automated

Reply:
thx, i'll give it a try